pkgs/build-support/libredirect/test.c

```diff
@@ -55,5 +55,5 @@ int main(void)
     /* If all goes well, this is never reached because test_execv() replaces
      * the current process.
      */
-    return 0;
+    return 1;
```

Comment:
This doesn't really matter here, because if the `execv` fails the program is aborted. I used `0` here in a way of saying "all of the tests ran though successfully" and we might want to use 0 again if we'd refactor `test_execv` to fork (hence it's not order-specific anymore).

Reply:
alright, I'll drop this commit before merging